touchforms/formplayer/api.py

```diff
@@ -321,35 +337,39 @@ def start_form_session(form_path, content=None, language="", session_data={}):
     """
     Start a new form session
     """
-    # TODO: this method has been deprecated and the config object 
+    # TODO: this method has been deprecated and the config object
     # should just be used directly. Temporarily left to support legacy code.
-    return XFormsConfig(form_path=form_path, 
+    return XFormsConfig(form_path=form_path,
                         instance_content=content,
                         session_data=session_data,
                         language=language).start_session()
-    
-def answer_question(session_id, answer, auth=None):
+
+
+def answer_question(session_id, answer, domain=None, auth=None):
     """
     Answer a question. 
     """
     data = {"action": "answer",
             "session-id": session_id,
-            "answer":answer }
-    return get_response(json.dumps(data), settings.XFORMS_PLAYER_URL, auth)
+            "answer": answer,
+            "domain": domain}
+    return get_response(json.dumps(data), auth)
 
-def current_question(session_id, auth=None):
+
+def current_question(session_id, domain=None, auth=None):
```

Comment:
@wpride @benrudolph 
It seems like the use of domain here is inconsistent - on the one hand it was required by the `post_data` function, but on the other hand it's an optional kwarg to all these functions and not all references to these functions were updated to pass it in. I think we should decide if domain should be required by `post_data` and if so we should update all references to these functions so that it's not an optional kwarg.